todo_app/tests_e2e/app_e2e_test.py

```diff
@@ -0,0 +1,99 @@
+import pytest
+import todo_app.app as app
+import todo_app.data.session_items as session
+from threading import Thread
+import requests, os
+from selenium import webdriver
+from dotenv import load_dotenv, find_dotenv
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+
+@pytest.fixture(scope='module')
+def test_app():
+# Create the new board & update the board id environment variable    
+    file_path = find_dotenv('.env')
+    load_dotenv(file_path, override=True)
+    board_id = session.create_trello_board()
+    os.environ['TRELLO_BOARD_ID'] = board_id
+    # construct the new application
+    application = app.create_app()
+    # start the app in its own thread.
+    thread = Thread(target=lambda: application.run(use_reloader=False))
+    thread.daemon = True
+    thread.start()
+    yield app
+    # Tear Down
+    thread.join(1)
+    session.delete_trello_board(board_id)
+
+@pytest.fixture(scope='module')
+def test_app_driver():
+
+    file_path = find_dotenv('.env')
+    load_dotenv(file_path, override=True)
+    # construct the new application
+    application = app.create_app()
+    # start the app in its own thread.
+    thread = Thread(target=lambda: application.run(use_reloader=False))
+    thread.daemon = True
+    thread.start()
+    yield app
+    # Tear Down
+    thread.join(1)
+   
```

Comment:
What's the point of this fixture? Delete it and just use `test_app`

If you were worried about recreating the board, `@pytest.fixture(scope='module')` means it won't have to re-run within the module.